File: 3790-fruits-into-baskets-ii/fruits-into-baskets-ii.ts
function numOfUnplacedFruits(fruits: number[], baskets: number[]): number {
    let output: number = 0;

    for(let fruit of fruits){
        let unplaced: number= 1;
        for(let i=0;i<=baskets.length-1; i++){
            if(fruit<= baskets[i]){
                unplaced= 0;
                baskets[i]=0;
                break;
            }
        }
        output= output+unplaced;
    }
    return output
};

File: 3790-fruits-into-baskets-ii/fruits-into-baskets-ii.py
class Solution(object):
    def numOfUnplacedFruits(self, fruits, baskets):
        """
        :type fruits: List[int]
        :type baskets: List[int]
        :rtype: int
        """
        output = 0
        for fruit in fruits:
            unplaced = 1
            for i in range(len(baskets)):
                if fruit <= baskets[i]:
                    unplaced = 0
                    baskets[i]=0
                    break
            output+=unplaced
        return output